clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonServerGroup.groovy

```diff
@@ -132,9 +147,27 @@ class AmazonServerGroup implements ServerGroup, Serializable {
     return securityGroups
   }
 
+  /**
+   * Get a single instance type for a server group.
+   * Used for the case of single instance type i.e. ASG with launch config, launch template, mixed instances policy without overrides
+   *
+   * For the case of multiple instance types i.e. ASG MixedInstancesPolicy with overrides, gather types from mixedInstancesPolicy.
```

Comment:
Would love to hear if you can think of a better way to handle this inconsistency.